packages/backend/src/db/PostgreSQLBadgeHubMetadata.ts

```diff
@@ -426,15 +439,55 @@ and v.app_metadata->'badges' @>
         ${filter.slugs[0]}`;
       } else {
         query = sql`${query}
-      and p.slug = any(${filter.slugs})`;
+      and p.slug = any(
+        ${filter.slugs}
+        )`;
       }
     }
 
     if (filter.search) {
       const matcher = `%${filter.search.toLowerCase()}%`;
       query = sql`${query}
-                    and (v.app_metadata->>'name' ilike ${matcher} or v.app_metadata->>'description' ilike ${matcher} or p.slug like ${matcher})
-                    or exists (select 1 from project_latest_categories plc where plc.project_slug = p.slug and plc.category_name ilike ${matcher})`;
+                    and (v.app_metadata->>'name' ilike
```

Comment:
This looks a bit odd

Reply:
Indeed, that's an unintentional Intellij Autoformat, thanks the catch!